static/src/stylesheets/module/content-garnett/_types.scss

```diff
@@ -696,3 +690,23 @@ $quote-mark: 35px;
         // only way to override defaults
     }
 }
+// *************** Feature showcase overrides ***************
+.content--type-feature {
+    .media-primary--showcase {
+        margin: 0 auto!important !important;
```

Comment:
oh my word...

Reply:
how about a `.media-primary--feature-showcase` class to avoid that important? woudl that work?

Reply:
have fixed this, was due to an earlier change. no more importants (or double importants)